### PR TITLE
issue_5 added more logging and exception handling

### PR DESCRIPTION
**Pull request by zqian**

### CODE REVIEW

File: kaltura_admin.py [hunk 1]
```diff
@@ -53,21 +55,26 @@ def get_kmc_admin_list(kaltura_user_id, kmcs):
...
+                    kmc_user_id = item.id.lower()
+                    if '@' in kmc_user_id:
+                        # remove @xxxxx
+                        item_id = kmc_user_id.partition('@')
+                        id = item_id[0]
+                        kmc_user_id = id
+
+                    # add to user_id_list
+                    user_id_list.append(kmc_user_id)
+        except KalturaException as ke:
```
Comment (jonespm) [inline kaltura_admin.py:73]: Probably don't need the more specific `KalturaException` unless you were doing something different here. The message looks the same.

File: kaltura_admin.py [hunk 2]
```diff
@@ -53,21 +55,26 @@ def get_kmc_admin_list(kaltura_user_id, kmcs):
...
+            log.error(f'Problem retrieving items for kmc: {kmc_name} {type(ke)} {ke.args} {ke}')
+        except Exception as e:
+            log.error(f'Problem retrieving items for kmc: {kmc_name} {type(e)} {e.args} {e}')
```
Comment (jonespm) [inline kaltura_admin.py:76]: Could use `log.exception` to still keep the traceback in here and see which like the exception occurred on.

**Review by jonespm — APPROVED** [review]

File: kaltura_admin.py (hunk 1 above)
Comment (zqian) [reply kaltura_admin.py]: Done. Remove the block for KalturaException

File: kaltura_admin.py (hunk 2 above)
Comment (zqian) [reply kaltura_admin.py]: Done. Changed into log.exception

**Pull request merged**